src/main/webapp/ts/profileBuilder.ts

```diff
@@ -1,39 +1,67 @@
-type authInfoFromServlet = {
+type authInfo = {
     loginUrl: string;
     logoutUrl: string;
     hasProfile: boolean;
 };
 
 function setForm(value: string) {
-    const menteeForm =  document.getElementById('Mentee');
-    const mentorForm = document.getElementById('Mentor');
+
     const userTypeButton = document.getElementById('user-input');
+    const menteeInputs: string[] = ['name-input', 'school-input', 'major-input'];
+    const mentorInputs: string[] = ['profName-input', 'company-input', 'careerTitle-input', 'bio-input', 'specialty-input']
 
-    if(menteeForm && mentorForm && userTypeButton){
-        if(value == 'Mentee') {
-            menteeForm.setAttribute('style','display:block;');
-            mentorForm.setAttribute('style','display:none;');
-            userTypeButton.setAttribute('value','Mentee');
+    if (userTypeButton) {
```

Comment:
L13: Could you add a console error on failure?
In fact, a cleaner approach would be to do something like `if (userTypeButton == null) ... console error and return ...` right under L9, that way you avoid creating a whole code-block for this.

Reply:
Done